src/widgets/setupdialog.cpp

```diff
@@ -230,6 +230,8 @@ bool SetupDialog::validateAdvancedSetupPage()
         emit getTarsnapVersion(_tarsnapDir);
 
     _ui.advancedPageProceedButton->setEnabled(result);
+    if (result)
+	    _ui.advancedPageProceedButton->setFocus();
```

Comment:
Are you sure about this? Again I don't like the fact that the focus is stolen away from the line edit mid typing, this is very confusing. To get the idea how weird this can get try pasting this "/usr/local/bin////" and then attempt to delete the extra slashes using the backspace key. For each keystroke your focus is stolen away in favor of the proceed button.